bot.py

```diff
@@ -112,13 +124,119 @@ def on_message(ws, message):
         if admin_message(response):
             return
 
-    elif "team_join" == response["type"]:
-        if NEWUSERGREETING:
-            new_user_message(response)
-        USER_LIST = sc.api_call(  # noqa: 841
-            "users.list",
-        )
 
+    #TODO: Purge all this shit
+    # elif "team_join" == response["type"]:
+    #     if NEWUSERGREETING:
+    #         new_user_message(response)
+    #     USER_LIST = sc.api_call(  # noqa: 841
+    #         "users.list",
+    #     )
+
+    log(response)   
+
+def log(response):
+    global engine
+    def add_user(r):
+        q = engine.execute("SELECT last_name FROM users WHERE slack_id=%s", r["user"])
+        if q.fetchall() == []:
+            r = sc.api_call(
```

Comment:
You know the type of information this response will give you correct? If so you could name it `(whatever type of info this request gives you)` or `(whatever type of info this request gives you)_response`